CHANGELOG.adoc

```diff
@@ -0,0 +1,15 @@
+= Asciidoctor Chart Changelog
+:icons: font
+:uri-repo: https://github.com/asciidoctor/asciidoctor-chart
+
+This document provides a high-level view of the changes introduced in Asciidoctor Chart by release.
+For an even more detailed look at what has changed, refer to the {uri-repo}/commits/[commit history] on GitHub.
+
+The format is based on https://keepachangelog.com/en/1.0.0/[Keep a Changelog],
+and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Versioning].
+
+== Unreleased
+
+=== icon:archive[] Housekeeping
+
+* Introduce a changelog https://github.com/asciidoctor/asciidoctor-chart/issues/30[(#30)]
```

Comment:
I think it is better to full qualify the issue, so if we publish to Antora or some other kind of documentation we have the full link to the issue and not only on GitHub release page.

Reply:
We can use an Asciidoctor extension if we want to automatically add a link. I don't want to add too much burden on the person who will maintain the changelog.

Reply:
Ok fine you're right, I forgot this ;)